examples/data-objects/key-value-cache/src/index.ts

```diff
@@ -88,8 +88,8 @@ class KeyValue implements IKeyValue, IFluidObject, IFluidRouter {
         return defaultFluidObjectRequestHandler(this, request);
     }
 
-    private async initialize() {
-        if (!this.runtime.existing) {
+    private async initialize(existing: boolean) {
+        if (!existing) {
```

Comment:
I'm not sure it's the right time to consider it, but worth sharing:
I'm looking more and more at creation of things as declaration of schema. I.e. if we look at fluid objects as one big graph, types of DDSs / Data stores declare the meaning / interpretation of data (and ops) in some sub-sections of this graph. Creation of DDS is essentially a declaration of schema for that part of container graph.
And logical continuation here - concurrent non-conflicting schema declaration for same node should be allowed and it should coalesce. I.e. it should be fine to create a map with this name if it is already there - it should be a noop.
This direction works perfectly if initialization is empty, i.e. like here - we do not put any content into the map on creation (otherwise initialization can overwrite user content that another client put into it).

So maybe most of our examples should not even use existing flag, and instead we should always "create" map, and make sure it does not explode on collision (probably needs to be opt-in behavior)